src/Abstractions/NexusMods.Abstractions.Collections/NexusCollectionLoadoutGroup.cs

```diff
@@ -0,0 +1,17 @@
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.NexusModsLibrary;
+using NexusMods.MnemonicDB.Abstractions.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.Collections;
+
+[Include<LoadoutItemGroup>]
+public partial class NexusCollectionLoadoutGroup : IModelDefinition
+{
```

Comment:
We should have a chat about this today. There are two types of collections:

1) Downloaded from Nexus Mods, these are supposed to be read-only
2) Created in the App, these are writable since the user is the "owner"

When creating a loadout we want to create a default collection named "My Collection". Everything the user does will go into this collection by default.

Not sure how we want to represent this in the DB, we can talk about that later.

Reply:
There are technically 2 ways to represent a collection.

1. Converting loadout to collection, i.e. Export
    - Same way you export PSD -> PNG
2. Treating a Loadout as a Collection (Suggested Here)

They have different semantics in terms of code.
The latter ensures loadouts will be easy to pack into collections, the former allows the App to remain more flexible.

Can go either way here.
Conceptually a loadout really is a collection, but what we have available in each format differs.
Maybe warrants a discussion.

Reply:
We can discuss after standup, but I think we can't have the loadout itself be the "My Collection", because from design, "My collection" is supposed to only contain the loose mods, not other installed collections, which would be at the same level as "My collection", not nested to it.

Reply:
Discussed at standup, I'll be adapting this post-merge to install in a format that fits into the UI and loadout system